cumulus/parachains/integration-tests/emulated/common/src/constants.rs

```diff
@@ -63,7 +63,7 @@ where
 
 /// Helper function to build the genesis storage using given json patch and code
 fn build_genesis_storage(patch: serde_json::Value, code: &[u8]) -> Storage {
-	let mut storage = GenesisConfigBuilderRuntimeCaller::new(code)
+	let mut storage = <GenesisConfigBuilderRuntimeCaller>::new(code)
```

Comment:
```suggestion
	let mut storage = GenesisConfigBuilderRuntimeCaller::new(code)
```

Reply:
this will not work.... (see other comment)